php_phongo.c

```diff
@@ -2518,12 +2551,13 @@ ZEND_INI_MH(OnUpdateDebug)
 		MONGODB_G(debug_fd) = NULL;
 	}
 
-	if (!new_value || (new_value && !ZSTR_VAL(new_value)[0])
-		|| strcasecmp("0", ZSTR_VAL(new_value)) == 0
-		|| strcasecmp("off", ZSTR_VAL(new_value)) == 0
-		|| strcasecmp("no", ZSTR_VAL(new_value)) == 0
-		|| strcasecmp("false", ZSTR_VAL(new_value)) == 0
-	   ) {
+	if (!new_value ||
+		(new_value && !ZSTR_VAL(new_value)[0]) ||
+		strcasecmp("0", ZSTR_VAL(new_value)) == 0 ||
+		strcasecmp("off", ZSTR_VAL(new_value)) == 0 ||
+		strcasecmp("no", ZSTR_VAL(new_value)) == 0 ||
+		strcasecmp("false", ZSTR_VAL(new_value)) == 0) {
+
```

Comment:
I noticed several other multi-line conditionals earlier in the PR that didn't include a blank line after `) {`. Were you going to go back and add them since you added a `clang-format` rule to preserve them?

Is there no way to have `clang-format` inject the newline on its own?